readthedocs-theme/templates/index.html

```diff
@@ -0,0 +1,113 @@
+{% extends "base.html" %}
+{# This is the blog index, not the home page template #}
+
+
+{% block content %}
+
+  <!-- Intro -->
+  <section class="ui vertical segment" id="blog-intro">
+    <div class="ui container">
+      <div class="ui segment basic">
+
+        {% block content_title %}
+          <div class="ui basic vertical big segment">
+          <h1 class="ui header">
+            <span class="ui header">Read the Docs Blog</span>
+          </h1>
+            <p>
+              Short intro
```

Comment:
I would just use one of the one-sentence intros we've used on other pages.

Reply:
This still needs changed. If you're not sure on what the copy is we're putting here it can be dropped.

Reply:
From the `company.html` copy:
```suggestion
              This is where we post all the week to week progress updates and announcements.
```

Reply:
Seems fine for now!